### ISSUE TITLE
Add some way to gradually decrease the learning rate

### ISSUE THREAD

**Issue by fredtcaroli** [NNBlocks/NNBlocks#6]

**Comment by fredtcaroli** [comment]
Now this can be done with custom procedures and the trainer.set_learning_rate(self, learning_rate) method.
